src/Common/src/Interop/Windows/Crypt32/Interop.certificates_imports.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Yep, Interop.certificates.cs and Interop.certificates_types.cs sounds good to me.